bundles/org.eclipse.search.core/search/org/eclipse/search/internal/core/text/IDirtyFileSearchParticipant.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.search.internal.core.text;
+
+import java.util.Map;
+
+import org.eclipse.core.resources.IFile;
+
+import org.eclipse.jface.text.IDocument;
+
+public interface IDirtyFileSearchParticipant {
```

Comment:
```suggestion
public interface DirtyFileProvider {
```

Reply:
I've renamed the interface as requested.